models/model_validation.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
```

Comment:
Can you write this as a notebook and provide the results of your model validation?